### PR TITLE
Optimization, removing code duplications

### PR DESCRIPTION
**Pull request by cevdetta**
- Optimize CSS
- Use lightningcss instead of postcss as CSS plugin
- Use exclude in deno.json
- Fix "color-scheme" meta tag

### CODE REVIEW

File: src/_includes/css/reset.css [hunk 1]
```diff
@@ -23,11 +20,7 @@ pre {
...
 button,
 input,
-optgroup,
 select,
 textarea {
-  font-family: inherit;
-  font-size: 100%;
-  line-height: 1.4;
-  margin: 0;
+  font: inherit;
```
Comment (oscarotero) [inline src/_includes/css/reset.css:25]: Why did you remove `optgroup`? And margin?

This code was taken [from modern-normalize](https://github.com/sindresorhus/modern-normalize/blob/main/modern-normalize.css), I guess it makes sense to be there. (You can see the comments of why these changes)

Comment (cevdetta) [reply src/_includes/css/reset.css:25]: I even requested it to be deleted: [modern-normalize#94](https://github.com/sindresorhus/modern-normalize/pull/94)
Because `optgroup` inherits the font features from `select` and I thought setting `margin: 0` would be unnecessary, because if someone sets margin on those form elements, it sure be overwritten by it, and setting "margin: 0" in the first place makes it worse.
If someone is going to change the margin -> it will be overwritten,
If not -> it will use browsers default.

Comment (oscarotero) [reply src/_includes/css/reset.css:25]: Fair. As long as this is just a basic boilerplate to create Lume themes, not intended to be a proper reset/normalizer, I'm okay. People will edit the CSS code as they want.

**Comment by oscarotero** [comment]
Thanks so much!
